### PR TITLE
prompts user with hidden password for login and proxy

### PR DESCRIPTION
**Pull request by nsauzede**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -20,7 +20,54 @@
 #include "quicky_exception.h"
 #include <iostream>
 
+#include <termios.h>	/* _getch */
+#ifndef WIN32
+#include <unistd.h>		/* getpass, getlogin */
```
Comment (quicky2000) [inline src/main.cpp]: Bad indentation on comment

File: src/main.cpp [hunk 2]
```diff
@@ -20,7 +20,54 @@
...
+
+    (void) tcsetattr(STDIN_FILENO, TCSANOW, &tty);
+#endif
+}
 
+void getpass2(std::string & p_passwd, const std::string & p_prompt)
+{
+    std::cout << p_prompt;
+    set_stdin_echo( false);
+	std::cin >> p_passwd;
```
Comment (quicky2000) [inline src/main.cpp]: Bad indentation

**Comment by nsauzede** [comment]
done

**Comment by quicky2000** [comment]
Thanks for your contribution and your efforts to write golden C++ code

**Pull request merged**